### PR TITLE
Chore(web): Add ImageBlock

### PR DESCRIPTION
**Pull request by atnagata**
# Overview
Add ImageBlock
https://deploy-preview-511--reearth-web.netlify.app/storybook/?path=/story/beta-components-blocks-imageblock--default

### CODE REVIEW

File: web/src/beta/components/Blocks/ImageBlock/index.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+export default {
+  component: ImageBlock,
+} as Meta;
+
+type Story = StoryObj<typeof ImageBlock>;
+
+export const Default: Story = {
+  args: {
+    src: "http://placehold.it/150X150",
```
Comment (KaWaite) [inline web/src/beta/components/Blocks/ImageBlock/index.stories.tsx:13]: we already have an public directory inside of .storybook with some sample photos. Please use those so we don't have to rely on an external API 🙏🏻

File: web/src/beta/components/Blocks/ImageBlock/index.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import { CSSProperties, FC, ImgHTMLAttributes } from "react";
+
+import { styled } from "@reearth/services/theme";
+
+type Props = {
+  src: NonNullable<ImgHTMLAttributes<HTMLImageElement>["src"]>;
+  align?: "left" | "center" | "right"; // default center
```
Comment (KaWaite) [inline web/src/beta/components/Blocks/ImageBlock/index.tsx:7]: You shouldn't need the comments here in the props.
Can you 
- remove the comments
- Inside the props destructuring(line 15) assign defaults (ie. `({src, align="center", fit="cover", .. .. . .. })`
- Lastly update the styled component's props to expect a value for fit and align always and remove the fallbacks

File: web/src/beta/components/Blocks/ImageBlock/index.tsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+import { styled } from "@reearth/services/theme";
+
+type Props = {
+  src: NonNullable<React.ImgHTMLAttributes<HTMLImageElement>["src"]>;
+  alt?: React.ImgHTMLAttributes<HTMLImageElement>["alt"];
+};
+
+const ImageBlock: React.FC<Props> = ({ src, alt }) => {
+  if (src) {
+    return (
```
Comment (KaWaite) [inline web/src/beta/components/Blocks/ImageBlock/index.tsx:11]: Like the Video block , please use a ternary operator with one return 🙇🏻

File: web/src/beta/components/Blocks/ImageBlock/index.tsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+      <Wrapper>
+        <ImageBox src={src} alt={alt} />
+      </Wrapper>
+    );
+  }
+  return (
+    <BlankImageBox>
+      <Icon icon={"image"} color={"#2e2e2e"} size={32} />
```
Comment (KaWaite) [inline web/src/beta/components/Blocks/ImageBlock/index.tsx:19]: Don't need { }
so `icon="image" color="#2e2e2e"`

File: web/src/beta/components/Blocks/ImageBlock/index.tsx [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    </BlankImageBox>
+  );
+};
+
+const Wrapper = styled.div`
+  position: absolute;
```
Comment (KaWaite) [inline web/src/beta/components/Blocks/ImageBlock/index.tsx:25]: Don't think you need this

File: web/src/beta/components/Blocks/ImageBlock/index.tsx [hunk 6]
```diff
@@ -0,0 +1,43 @@
+import Icon from "@reearth/beta/components/Icon";
+import { styled } from "@reearth/services/theme";
+
+type Props = {
+  src: NonNullable<React.ImgHTMLAttributes<HTMLImageElement>["src"]>;
+  alt?: React.ImgHTMLAttributes<HTMLImageElement>["alt"];
```
Comment (KaWaite) [inline web/src/beta/components/Blocks/ImageBlock/index.tsx:6]: You don't need to type so specifically. For some elements, where there are specific values needed, this is good.
But for something like src and alt, any string is okay, so you can just type these as `string`

**Review by isoppp — APPROVED** [review]

File: web/src/beta/components/Blocks/ImageBlock/index.tsx (hunk 2 above)
Comment (FabPARKJAESUNG) [reply web/src/beta/components/Blocks/ImageBlock/index.tsx]: The code has changed.
Above code no longer exists.

**Pull request closed without merging**